qaframework-bdd-tests/src/test/resources/features/refapp-2.x/14-clinicalVisit/clinicalVisit.feature

```diff
@@ -47,6 +47,12 @@ Feature: Clinical Visit Management
     And a user attaches patient supporting file
     And a user clicks the upload file button
     Then the system uploads the file
+    # User story: Use of clear forms button while attaching document
+    When a user clicks on Attachments link from patient visits dashboard
+    Then the system loads Attachments page
+    And a user attaches patient supporting file
```

Comment:
what happens when the user attaches a file?

Reply:
> what happens when the user attaches a file?

@mherman22 did you have a look at the next step on line 54?

Reply:
confusion is on me, i confused `clear forms` for an action that happens after clicking the upload button.

Reply:
> confusion is on me, i confused `clear forms` for an action that happens after clicking the upload button.

Have a cup of some hot ginger and lemon. :-)

Reply:
@mherman22 , @sherrif10 , @kdaud  are there any more reviews on this issue? Let me make a commit to trigger a build and see if the firefox tests will pass.

Reply:
none from me
cc: @sherrif10 @kdaud

Reply:
@Irenyak1  Fix the visibility of elements that are breaking the visit note feature file.